src/helm/common/critique_request.py

```diff
@@ -33,8 +43,18 @@ class CritiqueTaskTemplate:
     name: str
     """Name of the template."""
 
+    description: str
+    """Human-friendly short description that will be displayed to the workers on the index page.
+
+    Only used by Mechanical Turk."""
+
+    reward: str
```

Comment:
If we specify `description` and `reward` here, how do we also specify worker requirements? Are these fields being used?

Reply:
Removed (not used any more).

The worker requirements are specified in the web UI.